yellowbrick/classifier/prcurve.py

```diff
@@ -336,12 +349,22 @@ def _draw_multiclass(self):
         """
         # TODO: handle colors better with a mapping and user input
```

Comment:
I think we can remove this comment now :)

Reply:
Done.